metaswitch/common/test/alarms_parser.py

```diff
@@ -32,53 +32,59 @@
 
 import unittest
 from metaswitch.common.alarms import CLEARED, CRITICAL
-from metaswitch.common.alarms_parser import parse_alarms_file, render_alarm
+from metaswitch.common.alarms_parser import parse_alarms_file, render_alarm, alarms_to_dita
 
 class AlarmsParserTestCase(unittest.TestCase):
     def testValidFile(self):
         alarms = parse_alarms_file('metaswitch/common/test/test_valid_alarms.json')
         test_alarm = alarms[0]
-        self.assertEqual(test_alarm[0], 'NAME', msg="Incorrect name.")
-        self.assertEqual(test_alarm[1], 1000, msg="Incorrect index.")
-        self.assertIn(CLEARED, test_alarm[2], msg="No cleared state.")
-        self.assertIn(CRITICAL, test_alarm[2], msg="No critical state.")
+        self.assertEqual(test_alarm._name, 'NAME', msg="Incorrect name.")
+        self.assertEqual(test_alarm._index, 1000, msg="Incorrect index.")
+        self.assertIn(CLEARED, test_alarm._levels.keys(), msg="No cleared state.")
+        self.assertIn(CRITICAL, test_alarm._levels.keys(), msg="No critical state.")
+        self.assertEqual(render_alarm(test_alarm), 'NAME = (1000, 1, 3)\n')
```

Comment:
Can you not move the render_alarm test into these UTs and keep them separate (so as to not have one big UT)?